doc/guide/Check.md

```diff
@@ -4,6 +4,10 @@ summary: Check control guide
 ---
 
 # Check
+
```

Comment:
should provide the migration information. e.g Use `Xamarin.Forms.CheckBox` or `Xamarin.Forms.Switch` instead.